PlayFabClientSDK/source/PlayFabTransport.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PlayFab.Internal;
+
+namespace PlayFab
+{
+    /// <summary>
+    /// Sends request data.
+    /// </summary>
+    public class PlayFabTransport: ITransportPlugin
+    {
+        private static readonly IPlayFabHttp _http;
+
+        static PlayFabTransport()
+        {
+            var httpInterfaceType = typeof(IPlayFabHttp);
+            var types = typeof(PlayFabHttp).GetAssembly().GetTypes();
+            foreach (var eachType in types)
```

Comment:
Just so I'm clear this basically sets the expectation that the user has one concrete implementation of IPlayFabHttp in their assembly. What if the user has more than one IPlayFabHttp implementation? It looks like this arbitrarily picks one and doesn't let the user change it.

Reply:
Victor and I were just talking about that. There will be a slight modification in just a bit.

Reply:
I just talked to Paul about this at the same time as you commented. This will be changed. Thanks!

---
In reply to: [201480702](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r201480702) [](ancestors = 201480702)

Reply:
That file should actually be dropped, it hang around by accident. Thanks for finding it!

---
In reply to: [201490437](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r201490437) [](ancestors = 201490437,201480702)